src/safeds_stubgen/api_analyzer/_ast_visitor.py

```diff
@@ -95,13 +95,22 @@ def enter_moduledef(self, node: MypyFile) -> None:
             elif isinstance(definition, ExpressionStmt) and isinstance(definition.expr, StrExpr):
                 docstring = definition.expr.value
 
+        # Create module id to get the full path
+        module_id_parts = node.fullname.split(self.api.package)[1:]
+        module_id = self.api.package.join(module_id_parts)
+        if module_id.startswith("."):
+            module_id = module_id[1:]
+        module_id = module_id.replace(".", "/")
+        formatted_module_id = f"{self.api.package}/{module_id}"
+        id_ = self.__get_id(formatted_module_id)
```

Comment:
What does `__get_id` do? Please add a docstring. In any case, passing an ID to a `__get_id` function and getting an ID back seems weird.

Reply:
Also: A single underscore suffices to mark a method as internal, i.e. `_get_id`. A double underscore prefix does some magic on the names of class methods.

Reply:
Added a docstrings, renamed the ```__get_id``` function and adjusted the function.